superset-frontend/src/views/CRUD/data/database/DatabaseModal/DatabaseConnectionForm.tsx

```diff
@@ -115,78 +121,90 @@ const CredentialsInfo = ({
             placeholder="Paste content of service credentials JSON file here"
           />
           <span className="label-paste">
-            {t('Copy and paste the entire service account .json file here')}
+            {t(
+              'Copy and paste the entire service account .json file here to enable connection',
+            )}
           </span>
         </div>
       ) : (
-        <div
-          className="input-container"
-          css={(theme: SupersetTheme) => infoTooltip(theme)}
-        >
-          <div css={{ display: 'flex', alignItems: 'center' }}>
-            <FormLabel required>{t('Upload Credentials')}</FormLabel>
-            <InfoTooltip
-              tooltip={t(
-                'Use the JSON file you automatically downloaded when creating your service account in Google BigQuery.',
-              )}
-              viewBox="0 0 24 24"
-            />
-          </div>
-
-          {!fileToUpload && (
-            <Button
-              className="input-upload-btn"
-              onClick={() => document?.getElementById('selectedFile')?.click()}
-            >
-              {t('Choose File')}
-            </Button>
-          )}
-          {fileToUpload && (
-            <div className="input-upload-current">
-              {fileToUpload}
-              <DeleteFilled
-                onClick={() => {
-                  setFileToUpload(null);
-                  changeMethods.onParametersChange({
-                    target: {
-                      name: 'credentials_info',
-                      value: '',
-                    },
-                  });
-                }}
+        db?.engine !== 'gsheets' && (
```

Comment:
```suggestion
        db?.engine === 'bigquery' && (
```

We don't want to risk showing the copy below ('Use the JSON file you automatically downloaded when creating your service account in Google BigQuery.') for non-BQ databases.